src/components/lineChart.js

```diff
@@ -23,7 +23,13 @@ class Chart extends React.Component {
       .then((result) => {
 
         let dpJson = result.data
-        let dp = new DataPackage(dpJson)
+        let dp
```

Comment:
Is dp a Data Package or a view or ...? If so I think this whole section might read better as:

```
// this should create DP object and do not much else
var dp = new DataPackage(dpJson, dataPackageUrl);
// we can then have methods like ...
var dataPackageView = dp.views[idx]
// which return a DataPackageView object with methods like ...
dataPackageView.raw // raw descriptor
dataPackageView.vegaSpec
dataPackageView.plotlySpec

// this is then used in our view factory (not sure this is the right pattern - but something like it)
var myVegaView = VizViewFactory.makeView(dataPackageView)

// PS: I think we want to rename view to viz in data package (view is overloaded)
```

Writing this I think we should chat about it.